splash.xml

```diff
@@ -1,30 +1,53 @@
 <theme>
+
+   <!--
+   ///
+   Variables
+   ///
+   -->
+   <variables>
+      <distributionPath ifSubset="distribution:retrobat">~/.emulationstation/</distributionPath><!-- RetroBat\emulationstation\.emulationstation\ -->
+      <distributionPath ifSubset="distribution:batocera">/userdata/</distributionPath><!-- /userdata -->
+      <themeCustomizationsPath>${distributionPath}theme-customizations/art-book-next/</themeCustomizationsPath>
+   </variables>
+   
+   <include>./colors.xml</include>
+   <include ifSubset="color-scheme:custom">${themeCustomizationsPath}colors.xml</include>
+
    <formatVersion>7</formatVersion>
    <view name="splash">
       <splash name="splash">
-         <backgroundColor>000000</backgroundColor>
+		 <backgroundColor>${splashBackgroundColor}</backgroundColor>
       </splash>
-      <image name="background">
+      <image ifSubset="splash-screen-background:custom" name="background">

```

Comment:
please change lines 22 to 33 to this:
```
      <image ifSubset="splash-screen:custom" name="background">
         <origin>0.5 0.5</origin>
         <pos>0.5 0.5</pos>
         <minSize>1 1</minSize>
         <path>${themeCustomizationsPath}splash.png</path>
         <path>${themeCustomizationsPath}splash.jpg</path>
      </image>
      <image ifSubset="splash-screen:default" name="background">
         <origin>0 0</origin>
         <pos>1 1</pos>
      </image>
```
(please also make sure its uses spaces instead of tabs and spaces should be "3" - just to make it consistent with the rest of the file formatting)